Make code snippets longer than one line fenced code blocks (#179)
codatio/codat-docs: docs/assess/portal/categorization-of-accounts.md

The downloaded Excel file contains:

- The **Categorized Account Balances** tab.

> - The output of the endpoint below in <a href="/assess-api#/operations/get-data-companies-companyId-connections-connectionId-assess-accounts-categories">API reference</a> listing the categories.

Reviewer: ```suggestion
- The output from the <a href="/assess-api#/operations/get-data-companies-companyId-connections-connectionId-assess-accounts-categories">List suggested and confirmed account categories</a> endpoint.
```
